lib/moon/design/form/addon/btn.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Moon.Design.Form.Addon.Btn do
+  @moduledoc false
+
+  use Moon.StatelessComponent, slot: "default"
+
+  alias Moon.Design.Button
+
+  prop(attrs, :map, default: %{})
```

Comment:
No

Reply:
the same way i do plan to make a PasswordInput - It is supposed to use TextInput component & some new Addon.Eye